test/parallel/test-performance-measure-detail.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const util = require('util');
+const { performance, PerformanceObserver } = require('perf_hooks');
+
+const perfObserver = new PerformanceObserver((items) => {
+  items.getEntries().forEach((entry) => {
```

Comment:
Can you assert the length in here? If it is 0, this test will succeed, but it will never be called.

Reply:
Done !